draft-js-link-plugin/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "draft-js-link-plugin",
+  "version": "2.0.0-rc3",
+  "description": "Mention Plugin for DraftJS",
+  "author": {
+    "name": "Jan Amann",
+    "email": "jan@amann.me",
+    "url": "http://amann.me"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/draft-js-plugins/draft-js-plugins.git"
+  },
+  "main": "lib/index.js",
+  "keywords": [
+    "editor",
+    "wysiwyg",
+    "draft",
+    "react",
+    "ux",
+    "components",
+    "widget",
+    "link",
+    "react-component"
+  ],
+  "scripts": {
+    "clean": "../node_modules/.bin/rimraf lib",
+    "build": "npm run clean && npm run build:js && npm run build:css",
+    "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ../node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
+    "build:css": "node ../scripts/concatCssFiles $(pwd) && ../node_modules/.bin/rimraf lib-css",
+    "prepublish": "npm run build"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "decorate-component-with-props": "^1.0.2",
+    "immutable": "~3.7.4",
+    "prepend-http": "1.0.4",
+    "prop-types": "^15.5.8",
+    "union-class-names": "^1.0.0",
+    "url-regex": "4.1.1"
```

Comment:
I've added them to the main package.json now. Looks like on CI only those dependencies are installed.